AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/Support/Helpers/DeviceMetadataHelper.swift

```diff
@@ -31,4 +32,17 @@ struct DeviceMetadataHelper {
             return .noData
         }
 
+    static func removeDeviceMetaData(
+        of username: String,
```

Comment:
nit: naming discrepancy b/t get & remove: `getDeviceMetadata(for username:, with environment:)` vs `removeDeviceMetadata(of username:,environment)`

Reply:
Updated the `removeDeviceMetaData` to match